doc/rdesktop.1

```diff
@@ -123,6 +128,11 @@ but everything after this is unencrypted (including interactive logins).
 Do not send mouse motion events.  This saves bandwidth, although some Windows
 applications may rely on receiving mouse motion.
 .TP
+.BR "-M"
```

Comment:
Probably so, but Win10 mouse was completely broken without it. I don't like intermediate steps I can't test.